multiqc/modules/interop/interop.py

```diff
@@ -195,137 +206,198 @@ def run_metrics_summary_table(self, data):
-        }
-        headers['%>=Q30'] = {
-            'title': '%>=Q30',
-            'description': 'The percentage of bases with a quality score of 30 or higher, respectively.'
-        }
-        headers['Yield'] = {
-            'title': 'Yield',
-            'description': 'The number of bases sequenced which passed filter.'
+            'description': 'The number of clusters (in millions) passing filtering.',
+            'suffix': 'M',
         }
         headers['Cycles Error'] = {
             'title': 'Cycles Error',
             'description': 'The number of cycles that have been error-rated using PhiX, starting at cycle 1.'
         }
+        headers['Yield'] = {
+            'title': 'Yield',
+            'description': 'The number of bases sequenced which passed filter.',
+            'scale': 'PuOr'
+        }
         headers['Aligned'] = {
             'title': 'Aligned (%)',
-            'description': 'The percentage that aligned to the PhiX genome.'
+            'description': 'The percentage that aligned to the PhiX genome.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'PiYG'
         }
         headers['Error'] = {
             'title': 'Error Rate (%)',
-            'description': 'The calculated error rate, as determined by the PhiX alignment. '
+            'description': 'The calculated error rate, as determined by the PhiX alignment.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd'
         }
         headers['Error (35)'] = {
             'title': 'Error Rate 35 Cycles (%)',
-            'description': 'The calculated error rate for cycles 1-35.'
+            'description': 'The calculated error rate for cycles 1-35.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd',
+            'hidden': True
         }
         headers['Error (75)'] = {
             'title': 'Error Rate 75 Cycles (%)',
-            'description': 'The calculated error rate for cycles 1-75.'
+            'description': 'The calculated error rate for cycles 1-75.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd',
+            'hidden': True
         }
         headers['Error (100)'] = {
             'title': 'Error Rate 100 Cycles (%)',
-            'description': 'The calculated error rate for cycles 1-100.'
+            'description': 'The calculated error rate for cycles 1-100.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd',
+            'hidden': True
         }
         headers['Intensity C1'] = {
             'title': 'Intensity Cycle 1',
             'description': 'The corresponding intensity statistic at cycle 20 as a percentage of that value at the first cycle.',
             'help': 'equation: 100%x(Intensity at cycle 20)/(Intensity at cycle 1).'
```

Comment:
I don't think that `help` does anything?

Reply:
Also, I think that the description may be wrong - it looks to me like it should be _"The average of the A channel intensity measured at the first cycle averaged over filtered clusters."_

Reply:
I guess this text is taken from _% Intensity Cycle 20_ which isn't here?

Reply:
Also the other column with the same title ([here](https://github.com/matthdsm/MultiQC/blob/9435ac88ec3662931a5474f8ce33dce04d226c7d/multiqc/modules/interop/interop.py#L193-L196)) has an empty description...

Reply:
Right you are!